schema/schema.go

```diff
@@ -74,6 +74,11 @@ var searchContentDeletedEvent = `{
 	  "type": "string", 
 	  "default": ""
 	},
+	{
+      "name": "collection_id",
+      "type": "string",
+      "default": ""
+    },
```

Comment:
I notice other schemas use a compacted one-field-per-line style of definition that I think helps the readability. It's certainly not ideal to mix the two styles in the same file (see the `searchDataImportEvent` above).
NB also worth doing for othe fields above and below.
```suggestion
	{"name": "collection_id", "type": "string", "default": ""},
```